pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -600,6 +608,30 @@ pub mod pallet {
 				}
 			}
 		}
+
+		#[pallet::call_index(8)]
+		#[pallet::weight(10_000 + T::DbWeight::get().writes(1).ref_time())]
+		pub fn start_pack_messages(origin: OriginFor<T>, destination: Domain) -> DispatchResult {
+			let sender = ensure_signed(origin)?;
+
+			if PackedMessage::<T>::get((&sender, &destination)).is_none() {
+				PackedMessage::<T>::insert((sender, destination), T::Message::empty());
+			}
```

Comment:
`try_mutate` could be cleaner.

Reply:
I would also error out if the start has already happened once.

Reply:
Yeah, I was not sure about no-op and error out. But seeing that you also opted by erroring out, I will do so 👍🏻